packages/notifier/src/stored-notifier.js

```diff
@@ -5,6 +5,12 @@ import { makeSerializeToStorage } from '@agoric/internal/src/lib-chainStorage.js
 import { E, Far } from '@endo/far';
 import { observeNotifier } from './asyncIterableAdaptor.js';
 
+/**
+ * @import {ERef} from '@endo/far';
+ * @import {BaseNotifier, Notifier, StoredFacet} from './types.js';
+ * @import {Marshaller, StorageNode, Unserializer} from '@agoric/internal/src/lib-chainStorage.js';
```

Comment:
having `StorageNode` in `@agoric/internal` looks more and more awkward.

Reply:
yeah. and @agoric/notifier depending on ChainStorage. 
```
  "description": "Notifier allows services to update clients about state changes using a stream of promises",
```